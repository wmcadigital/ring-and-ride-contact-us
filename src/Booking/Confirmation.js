import Header from "../common/Header";
import BreadCrumb from "../common/BreadCrumb";
import FormContentWrapper from "../common/FormContentWrapper";
import ConfirmationPanel from "../common/ConfirmationPanel";
import Question from "../common/Question";
import CustomerServiceDetails from "../common/CustomerServiceDetails";

const Confirmation = () => (
  <>
    <Header heading="Book a Ring and Ride journey" />
    <FormContentWrapper>
      <BreadCrumb currentPageName="Booking" />
      <ConfirmationPanel header="We've received your form" />
      <Question text="What happens next" />
      <p>
        You&apos;ll receive an email to confirm that we have received your 
        booking request.
      </p>
      <p>
        Journey requests need to be received by 2.30pm on the day prior to your preferred journey. 
        We will then email you to confirm your booking by 4pm that day.
      </p>
      <Question text="Give us a call" />
      <CustomerServiceDetails />
    </FormContentWrapper>
  </>
);

export default Confirmation;
