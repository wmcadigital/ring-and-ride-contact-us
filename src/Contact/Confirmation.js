import Header from "../common/Header";
import BreadCrumb from "../common/BreadCrumb";
import FormContentWrapper from "../common/FormContentWrapper";
import ConfirmationPanel from "../common/ConfirmationPanel";
import Question from "../common/Question";
import CustomerServiceDetails from "../common/CustomerServiceDetails";

const Confirmation = () => (
  <>
    <Header heading="Contact us" />
    <FormContentWrapper>
      <BreadCrumb currentPageName="Contact us" />
      <ConfirmationPanel header="We've received your enquiry" />
      <Question text="What happens next" />
      <p>Thank you for contacting Ring and Ride.</p>
      <p>
        We will acknowledge your query and keep you updated on progress until it
        is resolved.
      </p>
      <p>
        Our Customer Services Centre aim to respond to enquiries within 10
        working days. However, there may be occasions where this takes longer,
        as we may need to liase with a third party.
      </p>
      <p>
        If you have not received a response within 10 working days, please
        contact our Customer Services team.
      </p>
      <Question text="Give us a call" />
      <CustomerServiceDetails />
    </FormContentWrapper>
  </>
);

export default Confirmation;
