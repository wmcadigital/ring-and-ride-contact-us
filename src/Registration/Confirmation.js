import Header from "../common/Header";
import BreadCrumb from "../common/BreadCrumb";
import FormContentWrapper from "../common/FormContentWrapper";
import ConfirmationPanel from "../common/ConfirmationPanel";
import Question from "../common/Question";
import CustomerServiceDetails from "../common/CustomerServiceDetails";

const Confirmation = () => (
  <>
    <Header heading="Register for the Ring and Ride service" />
    <FormContentWrapper>
      <BreadCrumb currentPageName="Register" />
      <ConfirmationPanel header="We've received your form" />
      <Question text="What happens next" />
      <p>
        You&apos;ll receive an email to confirm that we have received your
        registration form.
      </p>
      <p>
        It can take up to 2 days to process a registration form. It may take
        longer if we need more information from you.
      </p>
      <p>
        If you have not received a response within 10 working days, please
        contact our Customer Services team.
      </p>
      <Question text="Give us a call" />
      <CustomerServiceDetails />
    </FormContentWrapper>
  </>
);

export default Confirmation;
