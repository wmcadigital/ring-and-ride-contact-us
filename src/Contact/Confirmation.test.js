import { createRenderer } from "react-test-renderer/shallow";

import Confirmation from "./Confirmation";

describe("Confirmation - Contact", () => {
  it("renders Confirmation for Contact Us correctly", () => {
    const renderer = createRenderer();

    renderer.render(<Confirmation />);

    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
