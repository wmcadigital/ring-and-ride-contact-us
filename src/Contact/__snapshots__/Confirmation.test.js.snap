// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confirmation - Contact renders Confirmation for Contact Us correctly 1`] = `
<React.Fragment>
  <Header
    heading="Contact us"
  />
  <FormContentWrapper>
    <BreadCrumb
      currentPageName="Contact us"
    />
    <ConfirmationPanel
      header="We've received your enquiry"
    />
    <Question
      text="What happens next"
    />
    <p>
      Thank you for contacting Ring and Ride.
    </p>
    <p>
      We will acknowledge your query and keep you updated on progress until it is resolved.
    </p>
    <p>
      Our Customer Services Centre aim to respond to enquiries within 10 working days. However, there may be occasions where this takes longer, as we may need to liase with a third party.
    </p>
    <p>
      If you have not received a response within 10 working days, please contact our Customer Services team.
    </p>
    <Question
      text="Give us a call"
    />
    <CustomerServiceDetails />
  </FormContentWrapper>
</React.Fragment>
`;
